htdocs/wp-content/themes/hollandskroon/src/Commands/PronamicIdealCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace HK\Commands;
+
+use \WP_CLI;
+use \WP_CLI\ExitException;
+
+class PronamicIdealCommand
+{
+	/**
+	 * @throws \JsonException
+	 * @throws ExitException
+	 */
+	public function __invoke($args, $assoc_args)
+	{
+		$paged = 1;
+		$posts_per_page = 100;
+
+		do {
+			// Get fulfilled payments older than 5 days in batches of 100
+			$payments = new \WP_Query(array(
```

Comment:
`array()` -> `[]` (is heel erg mierenneuken dit)
Als je een WordPress plugin zou bouwen zou dit wel goed zijn btw.